### ISSUE TITLE
[Feature Request]: RunInference Wrapper for Go SDK

### ISSUE THREAD

**Issue by riteshghorse** [apache/beam#23382]
### What would you like to happen?

Add a RunInference wrapper for Go SDK using Python External Transform.

- [x] Sklearn Inference Unkeyed ModelHandler
- [ ] Sklearn Inference Keyed ModelHandler
- [ ] PyTorch Inference with both keyed and unkeyed ModelHandler

### Issue Priority

Priority: 2

### Issue Component

Component: sdk-go

**Comment by riteshghorse** [comment]
.remove-label "awaiting triage"

**Comment by riteshghorse** [comment]
.take-issue

**Comment by riteshghorse** [comment]
Sklearn Inference Unkeyed ModelHandler is in for 2.45.0 along with the generic internal details required to wrap any other inference from Python SDK.  Rest of the wrappers are now target for 2.46.0 release.

**Comment by damccorm** [comment]
@riteshghorse since this isn't a release blocker I'm going to remove the 2.46 milestone. If we get to it in time we can always add it back
